provider/provider_yaml_test.go

```diff
@@ -548,7 +548,7 @@ type tagsTestStep struct {
 
 // TestAccDefaultTags tries to test all the scenarios that might affect provider defaultTags / resource tags
 // i.e. up, refresh, preview, import, etc
-func TestAccDefaultTags(t *testing.T) {
+func TestAccDefaultTagsWithImport(t *testing.T) {
```

Comment:
There is another test called TestAccDefaultTags in the repo and I was getting confused.